tests/testimages/double_sig/Dockerfile

```diff
@@ -1,12 +0,0 @@
-FROM gcc@sha256:f993601701a37bef71e7f8fc1ef9410b09b15556f4371b06dcc10202cc81f9ea as builder
```

Comment:
Why throw away the test image definition? I think it's good for docs and recreation/extension if we keep these

Reply:
i did put the files back under a new path: `test/testdata/dockerfiles/...`